src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
```

Comment:
Use 'for...of' loop instead of 'for...in' loop to iterate over the 'actions' array

Reply:
still not resolved